test/CMakeLists.txt

```diff
@@ -272,7 +272,10 @@ function(Spglib_add_test test)
 	endforeach ()
 
 	add_test(NAME ${ARGS_TEST_NAME}
-			COMMAND ${CMAKE_CTEST_COMMAND} --build-and-test ${CMAKE_CURRENT_SOURCE_DIR}/${test}
+			COMMAND ${CMAKE_COMMAND} -E env --modify PATH=path_list_prepend:${CMAKE_INSTALL_PREFIX}/lib
```

Comment:
This should be in test property to make debugging more legible

Reply:
Argh I commented on wrong PR again. I will unsubscribe here so I don't have both in notification